soda/cmd/validate/tag.go

```diff
@@ -0,0 +1,156 @@
+package validate
+
+import (
+	"go/ast"
+	"go/parser"
+	"go/token"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"github.com/pkg/errors"
+)
+
+type Tag struct {
+	name string
+	value string
+	structName string
+}
+
+func (t *Tag) getName() string  {
+	return t.name
+}
+
+func (t *Tag) getValue() string  {
+	return t.value
+}
+
+func (t *Tag) getStructName() string  {
+	return t.structName
+}
+
+func getPackages(folder string, models ...string) map[string]*ast.Package {
+	path := os.Getenv("GOPATH")
```

Comment:
You'll probably want to use `envy.GoPaths` instead, and remember you can have more than one `
GOPATH`.